package.json

```diff
@@ -16,5 +16,10 @@
     "client": "cd packages/client && yarn",
     "service": "cd packages/service && yarn",
     "format": "prettier -w packages/*/src/** packages/*/package.json tsconfig.json"
+  },
+  "dependencies": {
```

Comment:
we shouldn't be adding any of these dependencies to the root package.json. Put these in the `packages/service/package.json`